ehsm_kms_service/test/test_aes.py

```diff
@@ -0,0 +1,69 @@
+import requests 

```

Comment:
after the patch merged, you can submit another commit to do the following things:

1. rename it as test_kms_with_rest.py
2. following the same test flow(in the main.cpp) to refine the test cases:

test_aes_128(){
a. create cmk
b. encrypt a plaintext
c. decrypt the ciphertext and compare it's same with the plaintext.
}

test_generatedatakey(){
a. create cmk
b. generate a datakey with a keylen(e.g. keylen=16)
c. try to decrypt the cipher_datakey and compare it's same with plaint_datakey.
}

and later, you can add more test cases when you add the other interfaces(rsa_sign/rsa_verifiy, .....)